.github/workflows/codeql.yml

```diff
@@ -30,10 +30,10 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v1

```

Comment:
I believe this is missing the version number

Also, can we update the `analyze` step to v2 as well?